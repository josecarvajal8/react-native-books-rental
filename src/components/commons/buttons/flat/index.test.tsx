import React from 'react';
import { Text } from 'react-native-animatable';
import renderer from 'react-test-renderer';
import FlatButton from '.';
const mockProps = {
    children: () => <Text>{'hello'}</Text>,
    action: () => null
}
test('Render flatButton component correctly', () => {
    const flatButtonCmp = renderer.create(<FlatButton {...mockProps} />).toJSON();
    expect(flatButtonCmp).toMatchSnapshot();
});