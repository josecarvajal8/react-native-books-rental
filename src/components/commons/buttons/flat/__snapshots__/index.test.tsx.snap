// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render booksList component correctly 1`] = `
<RNGestureHandlerButton
  collapsable={false}
  onGestureEvent={[Function]}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
  onHandlerStateChange={[Function]}
  rippleColor={0}
>
  <View
    accessible={true}
    style={
      Object {
        "alignItems": "center",
        "height": 66.99776785714286,
        "justifyContent": "center",
        "opacity": 1,
        "width": 362.3188405797101,
      }
    }
  />
</RNGestureHandlerButton>
`;

exports[`Render flatButton component correctly 1`] = `
<RNGestureHandlerButton
  collapsable={false}
  onGestureEvent={[Function]}
  onGestureHandlerEvent={[Function]}
  onGestureHandlerStateChange={[Function]}
  onHandlerStateChange={[Function]}
  rippleColor={0}
>
  <View
    accessible={true}
    style={
      Object {
        "alignItems": "center",
        "height": 66.99776785714286,
        "justifyContent": "center",
        "opacity": 1,
        "width": 362.3188405797101,
      }
    }
  />
</RNGestureHandlerButton>
`;
